backend/run_dace.py

```diff
@@ -118,15 +119,16 @@ def get_property_metadata(force_regenerate=False):
                 if prop.choices is not None:
                     # If there are specific choices for this property (i.e. this
                     # property is an enum), list those as metadata as well.
-                    if (isinstance(prop, TypeClassProperty) and
-                        inspect.isclass(prop.choices) and
-                        issubclass(prop.choices, aenum.Enum)):
+                    if (isinstance(prop, TypeClassProperty)
```

Comment:
Please make sure the formatting changes in this file apart from your changed lines are reverted. Which formatting provider was used to achieve these changes, do you know? I may need to provide a yapf config to prevent this from accidentally happening.

Reply:
No worries - I should have included a yapf config a long time ago. Thanks! :-)